net-libs/telepathy-qt/telepathy-qt-0.9.8.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+inherit python-any-r1 cmake-utils virtualx
+
+DESCRIPTION="Qt bindings for the Telepathy D-Bus protocol"
+HOMEPAGE="https://telepathy.freedesktop.org/"
+SRC_URI="https://telepathy.freedesktop.org/releases/${PN}/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="debug farstream test"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
```

Comment:
You had to add `REQUIRED_USE` for `python-single-r1`, but with `python-any-r1` it is wrong.

Reply:
it seems you overlooked that comment - going to fix it for you together with the test dependencies on merge. ;)